css/components/button.css

```diff
@@ -0,0 +1,89 @@
+/* === css for button component === */
+
+button{
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    color: #fff;
+    padding: .7rem 1.3rem;
+    font-family: inherit;
+    font-size: var(--normal-font-size);
+    font-weight: var(--font-medium);
+    border: none;
+    border-radius: 5px;
+    width: 9rem;
+    margin-bottom: var(--mb-0-5);
+    cursor: pointer;
+}
+
+/* == basic button css  ==*/
+
+.btn-primary{
+    background-color: var(--primary-dark);
+}
+.btn-secondary{
+    background-color: var(--secondary);
+}
+.btn-danger{
+    background-color: var(--danger);
+}
+.btn-warning{
+    background-color: var(--warning);
+}
```

Comment:
Going by your image attached, the orange color is too light to differentiate from distance wrt to white. Choose a darker shade of orange

Reply:
sure will change that color.